recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -27,14 +31,22 @@
     "ollama pull granite-code:20b\n",
     "```\n",
     "\n",
+    "#### Install Ollama Server in the Notebook\n",
+    "\n",
+    "Following \"Option B: Shell Script (Linux)\" in [Getting Started with Ollama](https://github.com/ibm-granite-community/utils/blob/main/recipes/Getting_Started/Getting_Started_with_Ollama.ipynb), run the following commands in separate notebook cells:\n",
+    "\n",
+    "1. Install Ollama\n",
+    "2. Start the Ollama server\n",
+    "3. Pull down the Granite models\n",
+    "\n",
     "> **NOTE:** By default, this notebook only uses the `3b` model to ensure the widest set of users can run the code. This is also necessary for our CI/CD unit test environment. However, if you have access to a reasonably new and powerful machine, we recommend using the `8b` or the `20b` version instead.\n",
     ">\n",
     "> Change the value in the next cell if you decide to use the `8b` or `20b` model."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
I don't think you meant to commit this and all following changes.

Reply:
Good catch, not sure what happened there. Fixed now.